### DISCUSSION TITLE
Poweroff/Shutdown on Windows 10 does nothing

### DISCUSSION THREAD

**Discussion by a-super-admin** [AppGround-io/bluetooth-keyboard-and-mouse-support#313] [Bugs & Problems]
Hello, a have the pro version 5.4.6 of the Android Bluetooth keyboard app. Controlling Windows the red power off button under "Multimedia" has no functionality and nothing happens. On Android it will shutdown the device. Why does it not work under Windows? 
Greetz

**Comment by a-super-admin** [comment]
Is the "Multimedia" design only android optimised? When I create a custom remote, how can I send real windows commands like 'sys32/shutdown'?

**Comment by a-super-admin** [comment]
@AdamKane41?

**Comment by Grooves1210** [comment]
Keyboard macro to shutdown PC  Win+X   -->  U --> U
